dunwoody_disaster/views/action_selector.py

```diff
@@ -85,7 +85,10 @@ def selectRandom(self):
             and self.character.curStamina >= w.staminaCost
         ]
 
-        chosen_weapon = random.choice(weapons)
+        if len(weapons) > 0:
+            chosen_weapon = random.choice(weapons)
```

Comment:
Will this still allow enemies to use these default attacks if they do not need to use them?

Reply:
This is saying that the enemy will only use the default item if they can't use anything else.